samples/net/sockets/mt_http_server/README.rst

```diff
@@ -0,0 +1,62 @@
+.. _sockets-mt-http-server-sample:
+
+Socket Multithreaded HTTP Server
+################################
+
+Overview
+********
+
+The sockets/mt_http_server sample application for Zephyr implements a
+skeleton HTTP server using a BSD Sockets compatible API.
+This sample has similar functionality as :ref:`sockets-dumb-http-server-sample`
+except here we have support for multiple simultaneous connections and IPv6.
+Also the sample application has no compatibility with POSIX.
+This HTTP server example is very minimal and does not really parse an incoming
+HTTP request, just reads and discards it, and always serves a single static
+page.
+
+The source code for this sample application can be found at:
+:zephyr_file:`samples/net/sockets/mt_http_server`.
+
+Requirements
+************
+
+- :ref:`networking_with_host`
+- or, a board with hardware networking
+
+Building and Running
+********************
+
+Build the Zephyr version of the sockets/echo application like this:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/sockets/mt_http_server
+   :board: <board_to_use>
+   :goals: build
+   :compact:
+
+``board_to_use`` defaults to ``qemu_x86``. In this case, you can run the
+application in QEMU using ``make run``. If you used another BOARD, you
+will need to consult its documentation for application deployment
+instructions. You can read about Zephyr support for specific boards in
+the documentation at :ref:`boards`.
+
+After the sample starts, it expects connections at 192.0.2.1 or 2001:db8::1,
+port 8080. The easiest way to connect is by opening a following URL in a web
+browser: http://192.0.2.1:8080/ or http://[2001:db8::1]:8080/
+You should see a page with a sample content about Zephyr (captured at a
+particular time from Zephyr's web site, note that it may differ from the
+content on the live Zephyr site).
+Alternatively, a tool like ``curl`` can be used:
+
+.. code-block:: console
+
+    $ curl http://192.0.2.1:8080/
+
+Finally, you can run an HTTP profiling/load tool like Apache Bench
+(``ab``) against the server:
```

Comment:
if you end this line with a double colon, the next indented section will be treated as a code-block (which is your intention I think):
```
(``ab``) against the server::
```